jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/diagram/ProcessDiagramPresenter.java

```diff
@@ -61,9 +61,13 @@ public void onProcessSelectionEvent(@Observes final ProcessDefSelectionEvent eve
         String containerId = event.getDeploymentId();
         String serverTemplateId = event.getServerTemplateId();
         String processId = event.getProcessId();
-        processImageService.call((String svgContent) -> displayImage(svgContent)).getProcessDiagram(serverTemplateId,
-                                                                                                    containerId,
-                                                                                                    processId);
+        processImageService.call((String svgContent) -> displayImage(svgContent),
+                                 (String message, Throwable throwable) -> {
```

Comment:
I think in general we should avoid using the remote callback error like this because we might suppress other errors. 
Best would be handle it with the spec management call, testing if the container is started or not.